### DISCUSSION TITLE
Punycode

### DISCUSSION THREAD

**Discussion by Gorian** [CorentinTh/it-tools#1235] [Ideas]
Would be nice to add a punycode encoder/decoder

Example:
https://www.punycoder.com/

https://en.wikipedia.org/wiki/Punycode

**Comment by sharevb** [comment]
Hi @Gorian and @CorentinTh , implemented in #1258

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
